library/.gitignore

```diff
@@ -0,0 +1 @@
+/out/
```

Comment:
What's the reason for this addition?

Reply:
fixed https://github.com/Grover-c13/PokeGOAPI-Java/pull/945/commits/fbd834918a46c5e6bf45ff6285f0e62fea21b2b3

Reply:
You seem to have kept this, and instead removed it in the other .gitignore. Was this intended?

Reply:
I had added this because I was manually compiling the protos nothing much

Reply:
Remove this, and manually exclude anything you don't want.